src/coreclr/inc/readytorun.h

```diff
@@ -445,6 +447,8 @@ enum ReadyToRunHelper
     // Array helpers for use with native ints
     READYTORUN_HELPER_Stelem_Ref_I                = 0x113,
```

Comment:
`READYTORUN_HELPER_Stelem_Ref_I` and `READYTORUN_HELPER_Ldelema_Ref_I` are unused (left in by accident in #71571). Could you please delete them while you are on it?

Reply:
Addressed